src/en/roadmap.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Roadmap
+translationKey: roadmap
+layout: 'layouts/base.njk'
+eleventyNavigation:
+  key: roadmapEN
+  title: Roadmap
+  locale: en
+  order: 4
+  hideMain: true
+---
+
+# Roadmap
+## Timeline: Q1 (Apr - Jun 2024) 
+
+### Theme: Product Growth
+
+#### Items:
+- **Support for Server Side Rendering frameworks**
```

Comment:
@adorayi oh just noticed, should be mention that it's currently specific to React?
```suggestion
- **Support for React with Server Side Rendering frameworks**
```

Reply:
That was more of a general statement which the sub bullet expands on with details (NextJs). I don't think we need to change this now, we can revisit later. Thanks.